docs/DeveloperGuide.md

```diff
@@ -510,6 +529,50 @@ _{More to be added}_
 
     Use case ends.
 
+
+**Use case: Add a contact as favourite**
```

Comment:
I think this is fine since it's one particular case, but we can consider generalising and saying "Add contact(s)" rather than singular contact